COMPILING.md

```diff
@@ -26,6 +33,8 @@ On Ubuntu-based distributions 18.04+, Debian 9+, and Raspberry Pi OS:
 - qttools5-dev-tools
 - libjack-jackd2-dev
 
+**Note:** The exact dependencies might be different for older distributions. See [this comment by softins](https://github.com/jamulussoftware/jamulus/pull/2267#issuecomment-1022127426)
+
 ### On Fedora 33+
 
 - qt5-qtdeclarative-devel
```

Comment:
I'm wondering why qt5-qtdeclarative-devel should be the correct package. As far as I understand, it's mainly related to QML, which we don't use. Maybe it's a package which happens to pull in the right dependencies for us, but this should probably be re-checked.

Reply:
I don't know about Fedora. Someone else should have a look at that